docs/guide/usage/mqtt_topics_and_messages.md

```diff
@@ -629,3 +650,32 @@ See [Touchlink](./touchlink.md).
 #### zigbee2mqtt/bridge/request/touchlink/identify
 
 See [Touchlink](./touchlink.md).
+
+### Action
+
+#### zigbee2mqtt/bridge/request/action
+
+Allows to call specific pre-defined actions, usually manufacturer-specific.
+All action names are published in `zigbee2mqtt/bridge/definitions` under `actions`.
+
+::: tip
+Specific up-to-date actions/parameters can be observed directly in the source code [https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/converters/actions.ts](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/converters/actions.ts)
+:::
+
+##### Action: `hue_factory_reset`
+
+Allows to reset Hue devices via a manufacturer-specific Touchlink request.
+
+Parameters:
+
+- `extended_pan_id` - Hex string in `0x{id}` format - the extended PAN ID of the network the device(s) should try to join after reset. E.g. `0xa1b2c3d4e5f60123`
+- `serial_numbers` - Array of numbers - the serial numbers of the device(s) to reset. E.g. `[123456, 987654]`
```

Comment:
This might be slightly confusing. The serial numbers on hue stuff are 24 bit numbers printed in hex on the device. Here's a photo of a light strip from their FCC filing:
<img width="738" height="552" alt="image" src="https://github.com/user-attachments/assets/59ed8bed-4c62-4a17-86a8-cc25c0196184" />

The serial number in this case is `0x3A745C`, which in JSON would be `3830876`.

It might be worth mentioning that detail.